src/pages/index.js

```diff
@@ -146,12 +233,28 @@ addImageForm.setEventListeners();
 const previewImagePopup = new PopupWithImage("#preview-modal");
 previewImagePopup.setEventListeners();
 
+const confirmDeletePopup = new PopupConfirm("#delete-popup");
+confirmDeletePopup.setEventListeners();
+
+const avatarEditForm = new PopupWithForm(
+  "#profile-image-modal",
+  handleAvatarSubmit
+);
+avatarEditForm.setEventListeners();
+
+const avatarEditFormValidation = new FormValidator(
+  config,
+  profileEditImageForm
+);
+avatarEditFormValidation.enableValidation();
+
```

Comment:
Needs correction: I can't add likes to the new added cards. Also I can't remove the new added cards.

Reply:
Could you point me in the right direction on fixing this issue?

Reply:
You should check if you provide the id for the new cards